### DISCUSSION TITLE
Increase priority based on how long a station has not been visited by a train

### DISCUSSION THREAD

**Discussion by crackalackintv** [Opinionated-Blueprints/10-Books-Full-of-Rails#99] [Ideas]
I did make a circuit that increases the priority over time the longer the station has not been visited. I figured I share this here to see if this could be of use to the blueprint book.

https://www.reddit.com/r/factorio/comments/1gknhcn/dynamic_train_priority_adjuster_based_on_how_long/

**Comment by Bocian-1** [comment]
I don't see how this is useful.
A station may not have been visited for a long time simply because there was a lack of demand.
You'll get a bunch of trains going to a provider for enriched uranium simply because getting a full trainload takes a long time

**Reply by crackalackintv** [reply]
It only increases it when there is the ability to have a train arrive (Limit > 0). It makes it so trains don't just always prefer the closest station  but will reach further out stations as well if there is more demand than production. I think even with the new interrupts trains will still path to the closest station. ¯\_(ツ)_/¯

**Reply by Bocian-1** [reply]
I mean, sure, but I'm also a bit averse to adding complexity to solve problems that can be solved by simply 'more trains', and this can very much e solved by 'more trains' very easily. Trains are dirt cheap and with a push system you barely ever even need depots.
